src/test/java/racingCar/WinnersTest.java

```diff
@@ -0,0 +1,38 @@
+package racingCar;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class WinnersTest {
+
+    @Test
+    void 단일우승자(){
+        ArrayList<Car> carList = new ArrayList<>();
+        Car crong = new Car("crong",4);
+        carList.add(crong);
+        carList.add(new Car("pobi",3));
+        carList.add(new Car("honux",2));
+
+        Cars cars = new Cars(carList);
+        List<Car> winners = cars.winnerCars();
+
+        Assertions.assertThat(winners).contains(crong);
```

Comment:
```suggestion
        Cars cars = new Cars(Arrays.asList(
                new Car("crong", 4),
                new Car("pobi", 3),
                new Car("honux", 2)));

        Winners winners = new Winners(Arrays.asList(
                new Car("crong", 4)
        ));
        assertThat(cars.findWinners()).isEqualTo(winners);
```

이렇게 코드를 바꾸어도 테스트가 성공 할 수 있게 수정해보면 어떨까요?
지금은 Winners 객체도 테스트에서 사용해보면 좋을 것 같아요.